x-pack/plugins/ml/public/application/components/create_data_view_button/create_data_view_button.tsx

```diff
@@ -25,10 +26,9 @@ export const CreateDataViewButton = ({
     closeDataViewEditorRef.current = dataViewEditor?.openEditor({
       onSave: async (dataView) => {
         if (dataView.id && onDataViewCreated) {
-          onDataViewCreated(dataView.id, 'index-pattern', dataView.name);
+          onDataViewCreated(dataView);
```

Comment:
After creation of the new (adhoc or saved) data view, is it possible to update the Data view control with the newly created data view?

Reply:
@peteharverson it already happens, both with saved and ad-hoc data views

Reply:
done in https://github.com/elastic/kibana/pull/180795/commits/1d839bc3b965c1bc74ce898e02f4bc29d24d5dbe